src/test/java/bennett/rijksmuseum/RijksMuseumServiceTest.java

```diff
@@ -0,0 +1,63 @@
+package bennett.rijksmuseum;
+
+import bennett.rijksmuseum.json.CurrentCollection;
+import io.reactivex.rxjava3.core.Single;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class RijksMuseumServiceTest {
+
+    private static class TestRijksMuseumService implements RijksMuseumService {
```

Comment:
This is not how I told you to test services.

Reply:
I can't figure out how to do it properly, I rewatched the whole weather video but still couldnt get there. How should I do it?

Reply:
Use the factory and create a service...